msu/utils/globals.nut

```diff
@@ -148,3 +148,32 @@
 			return _object;
 	}
 }
+
+::MSU.deepEquals <- function(_a, _b)
```

Comment:
The original PR for this #339 was closed. I left [a comment](https://github.com/MSUTeam/MSU/pull/339#discussion_r1506845018) there that I think still applies. Please check reassess.

Reply:
Alright, thanks, will adjust based on that feedback.